docs/tutorials/dronecan_firmware_upgrade.rst

```diff
@@ -0,0 +1,134 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _dronecan_firmware_upgrade:
+
+*****************************
+DroneCAN Firmware Upgrade
+*****************************
+
+Necessary Applications/Hardware
+=================================
+
+Software
+----------
+In order to update your Vertiq module's firmware via DroneCAN, you must install the `DroneCAN GUI tool <https://dronecan.github.io/GUI_Tool/Overview/>`_. 
+Simply follow the instructions at the link provided in order to install the application.
+
+Hardware
+-----------
+In order to connect with the DroneCAN GUI tool, you will need an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_ which 
+will be used in each of the following examples. See the DroneCAN GUI tool's documentation to find more information on supported devices.
+
+DroneCAN GUI Tool Basics
+============================
+After opening the DroneCAN GUI tool, you are met with the following screen: 
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_gui_start_page.png
+
+Under *Select CAN Interface*, select the name of your connected USB-to-CAN device. In this case, the device is named USB Serial Device. 
+By default, all DroneCAN enabled Vertiq devices have a bitrate of 500000 bits per second. If you have not changed your module's bitrate, you must configure 
+*CAN bus bit rate* to 500000. Otherwise, you must set *CAN bus bit rate* to match the value set on your module. Select OK, and you will see the following window:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_gui_main_page.png
+    :scale: 40%
+
+Here, you can view other devices detected on the bus, configure devices, and importantly here, trigger firmware updates. In order to interact with other DroneCAN nodes on the bus, 
+the USB-to-CAN node needs its own unique node ID. To set your device's node ID, find *Set local node ID* in the top left, pick a node ID unique from all other devices on the 
+bus (the default 127 is unique from the default Vertiq node ID of 99), and click the check mark.
+
+Now, connect your module to your USB-to-CAN device, and power it on. You should see it appear in the list of *Online nodes*:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_seen.png
+
+Double click the module's node to find its DroneCAN node properties
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_properties.png
+    :scale: 80%
+
+From here, you can view the module's parameters by selecting *Fetch All*, you can reboot your module with *Restart*, or you can start a firmware update with *Update Firmware*. 
+In order to change a parameter's value, simply double click it, change the value, and click *Send*.
```

Comment:
Not really important, but you may be able to leave off this last sentence. We don't really care about teaching people to change parameters here, this is just about firmware upgrade, so might as well not give people other stuff to get distracted by.